scss/_variables.scss

```diff
@@ -959,6 +959,36 @@ $card-img-overlay-padding:          1.25rem !default;
 $card-group-margin:                 $grid-gutter-width / 2 !default;
 $card-deck-margin:                  $card-group-margin !default;
 
+// Accordion
+$accordion-padding-y:               1.25rem !default;
+$accordion-padding-x:               1.25rem !default;
+$accordion-bg:                      $white !default;
+$accordion-color:                   $body-color !default;
+$accordion-button-active-bg:        lighten($component-active-bg, 45%) !default;
+$accordion-button-active-color:     $primary !default;
+$accordion-button-padding-y:        $accordion-padding-y !default;
+$accordion-button-padding-x:        $accordion-padding-x !default;
+$accordion-body-padding-y:          $accordion-padding-y !default;
+$accordion-body-padding-x:          $accordion-padding-x !default;
+$accordion-button-color:            $accordion-color !default;
+$accordion-button-bg:               $accordion-bg !default;
+$accordion-border-width:            $border-width !default;
+$accordion-border-radius:           $border-radius !default;
+$accordion-border-color:            rgba($black, .125) !default;
+$accordion-icon-color:              $accordion-color !default;
+$accordion-icon-active-color:       $accordion-button-active-color !default;
+$accordion-icon-transition:         $transition-base !default;
+$accordion-button-icon-rotation:    0deg !default;
```

Comment:
Is the unit really needed here?

Reply:
Yeah not needed indeed, but like @MartijnCuppens said I will change the rotation to a generic transform property so the user can decide what he does with the icon instead of forcing them to use a rotation.